modin/config/__main__.py

```diff
@@ -42,18 +44,22 @@ def export_config_help(filename: str):
     filename : str
         Name of the file to export configs data.
     """
-    configs = pandas.DataFrame(
-        columns=[
-            "Config Name",
-            "Env. Variable Name",
-            "Default Value",
-            "Description",
-            "Options",
-        ]
-    )
+    configs_data = []
     for objname in sorted(globals()):
         obj = globals()[objname]
         if isinstance(obj, type) and issubclass(obj, Parameter) and not obj.is_abstract:
+            doc = obj.__doc__
+            description = []
+            indent = _get_indent(doc)
+            # remove indent
+            for line in doc.split("\n"):
+                if line:
+                    line = line[indent:]
+                description.append(line)
+
+            description = "\n".join(description)
```

Comment:
there's a function called [`dedent` in python's stdlib](https://docs.python.org/3/library/textwrap.html#textwrap.dedent) that does exactly the same:
```python
>>> txt = """
...     some string
...             with dedents
...     """
>>> print(txt)

        some string
                with dedents

>>> from textwrap import dedent
>>> print(dedent(txt))

some string
        with dedents

```

Reply:
Yep, `dedent` fits much better here.